packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.tsx

```diff
@@ -185,11 +185,30 @@ export class GuxAvatar {
     }
   }
 
+  private getUcIntegrationText(appName: string): JSX.Element | null {
+    switch (appName) {
+      case 'teams':
+        return 'Microsoft Teams';
+      case 'zoom':
+        return 'Zoom';
+      case '8x8':
+        return '8x8';
```

Comment:
I think this should be `return '8 by 8;' It will never be seen on screen and only readout by the screen reader.